server/include/ai/ai.h

```diff
@@ -0,0 +1,52 @@
+/*
+** EPITECH PROJECT, 2024
+** Zappy Server
+** File description:
+** AI struct
+*/
+
+#pragma once
+
+#include <stddef.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+#include "list/list.h"
+
+typedef struct s_app app_t;
+
+typedef struct s_ia {
+    size_t fd;
+    list_t *list_messages;
+    size_t level;
+} ia_t;
+
+/**
+ * @brief Create a AI object.
+ *
+ * @param app Application with list of team.
+ * @param fd File descriptor of AI.
+ * @param team Team of AI
```

Comment:
Don't forget the point at the end of the sentence.


```suggestion
 * @param team Team of AI.
```